.eslintrc.json

```diff
@@ -21,8 +21,7 @@
     "lodash": "readonly",
     "googlesitekitCurrentModule": "readonly",
     "gtag": "readonly",
-    "process": "readonly",
-    "wp": "readonly"
```

Comment:
Should this not remain `readonly`?

Reply:
Removing it entirely discourages the use of the `wp` global in anyway. There should be no reason to do that. And if you really need to, you can still use `const { wp } = window`